src/components/owned/OwnerInfo.tsx

```diff
@@ -72,39 +70,32 @@ function BadgeTitle({ badge }: { badge: BaseBadgeContract }) {
 }
 
 function BadgeContent({ badge }: { badge: BaseBadgeContract }) {
-  if (badge instanceof EmailBadgeContract) {
+  if (badge.type === 'Email') {
```

Comment:
Comparing to `string` is not 100% safe, maybe we can use `enums` for this? Just a suggestion, if you don't want to do it, you shoudn't :)